code_examples/core_features/package.json

```diff
@@ -15,6 +15,8 @@
     "dotenv": "^16.0.0"
   },
   "devDependencies": {
+    "node-fetch": "^2.6.7",
+    "@types/node-fetch": "2.6.2",
```

Comment:
`@types/node-fetch` seems to be pinned to an exact version. How do we make sure it does not break if `node-fetch` changes something? Is this something we should/could even take care of?

Reply:
oh, yeah that was not what I wanted, good catch